stress/common/src/main/java/alluxio/stress/common/MultipleNodeBenchSummary.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.stress.common;
+
+import alluxio.stress.Summary;
+import alluxio.stress.TaskResult;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * abstract class for summary which contains multiple node results.
+ *
+ * @param <T> the type of task result
+ */
+public abstract class MultipleNodeBenchSummary<T extends TaskResult> implements Summary {
+  protected Map<String, T> mNodeResults;
```

Comment:
you can initialize the map here already
```suggestion
  protected Map<String, T> mNodeResults = new HashMap<>();
```